src/routes/mod/[modId]/+page.svelte

```diff
@@ -141,7 +141,7 @@
             modName={$mod.data.mod.name}
             compatibility={$mod.data.mod.compatibility} />
         </div>
-        <ModLatestVersions modId={$mod.data.mod.id} latestVersions={$mod.data.mod.latestVersions} />
+        <ModLatestVersions modId={$mod.data.mod.mod_reference} latestVersions={$mod.data.mod.latestVersions} />
```

Comment:
Any particular reason this was changed to the reference?

Reply:
It was fixing the install in SMM button in the latest versions, SMM only accepts the mod reference to be passed to the URI. But I see now that this breaks the download buttons on the linked version page, because the API only accepts mod IDs in the download URL

Reply:
Well, in general we shouldn't be passing "mod_reference" to a field that is called "modId". If the rest of the downstream was renamed, I'd be fine with it.

Reply:
For now I've passed both to use in their respective fields, that sounds like a post-1.0 problem